### ISSUE TITLE
Who's maintaining?

### ISSUE THREAD

**Issue by Ry-K** [bmbsqd/elastic-identity#6]
Hi - I am wondering who is maintaining this now; either @icanhasjonas or @tstojecki (both)? I would like to fork from the one that is moving forward and:
- Update to the latest of all libs (outside of nunit, just latest of 2.6 due to the 3.0 rewrite).
- Add HTTP challenge and SSL support (optional accept all for dev use or add your own cert validation callback).
- Version/concurrency issue noted on this project.
- Add any needed tests.
- (fyi) ReSharper suggestions.

Note that if it's @icanhasjonas version, I can incorporate @tstojecki changes also. Let me know one way or the other.

Thanks in advance. Regards,
Ryan

**Comment by tstojecki** [comment]
hi @Ry-K 
@icanhasjonas let us know if you're interested in maintaining it going forward? I agree that at a minimum it needs to be regularly updated for the latest version of libs. 

If you're not interested, let us know if you're ok with maintaining it under my repo, or you would prefer some other way. We could just include a link on here that directs the users to that repo for future versions.

**Comment by tstojecki** [comment]
Thanks @icanhasjonas !

**Comment by icanhasjonas** [comment]
@tstojecki, tell me when :-)

**Comment by Ry-K** [comment]
Thanks for clearing this up @icanhasjonas and @tstojecki !

**Comment by tstojecki** [comment]
Hi @icanhasjonas 
We have made good progress on transitioning over to a new version thanks to @Ry-K.
I am going to need to figure out how we want to publish the package going forward. I was thinking we could rename and package to ElasticIdentity from BmbSqd.ElasticIdentity, but I wanted to make sure you would ok with that. That would mean that we would rename the root namespace to ElasticIdentity, rename the assemblies and update assembly info. I would reference your version in the readme and provide a history of how we arrived at the current version, including a link to your this repository.
Is that something you would be ok with?
